sale_partner_company_group/tests/test_sale_partner_company_group.py

```diff
@@ -10,16 +10,6 @@ class TestSalePartnerCompanyGroup(TestBasePartnerCompanyGroup):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        # Remove this variable in v16 and put instead:
-        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
```

Comment:
Weren't we supposed to import DISABLED_MAIL_CONTEXT ?
Is it coming from `TestBasePartnerCompanyGroup->BaseCommon` ?

Reply:
Yes, it is. https://github.com/OCA/partner-contact/blob/9e0fa464453ccad01c5b20a1b2f16613b3fedd96/base_partner_company_group/tests/test_base_partner_company_group.py#L7